### ISSUE TITLE
issue: reminder notification for already created video

### ISSUE THREAD

**Issue by stephanfriedrich** [KyleKun/one_second_diary#112]
hello everyone,
first of all, thank you for your user-friendly android app and all the work.

----

as a user, i would like to set a reminder notification for a specific time of day (e.g. 7pm) so that i receive a reminder if i have not yet created a video.

### issue description:

if i have already created a video at 12 noon, i still receive a "reminder notification" at my set time of day (7 pm).

### expectation

as a user, i expect not to receive a "notification" if a video has already been created.

best regards

**Comment by ishan-vaghani** [comment]
This is the limitations from library. As we are scheduling daily notifications there is no option to skip one particular day.

**Comment by ishan-vaghani** [comment]
The workaround is we need to setup notifications manually. For ex we need schedule notification for next 7 days. Whenever user opens the app again we can schedule notifications for next days. Let me know what do you think @KyleKun

**Comment by ishanvaghani** [comment]
This is the limitations from library. As we are scheduling daily notifications there is no option to skip one particular day.

**Comment by ishanvaghani** [comment]
The workaround is we need to setup notifications manually. For ex we need schedule notification for next 7 days. Whenever user opens the app again we can schedule notifications for next days. Let me know what do you think @KyleKun

**Comment by KyleKun** [comment]
> The workaround is we need to setup notifications manually. For ex we need schedule notification for next 7 days. Whenever user opens the app again we can schedule notifications for next days. Let me know what do you think @KyleKun 

Can it be done with more days?

Example: user opens app, record, notification gets disabled for today and activate for next 30 days.

Always disable the current day and schedule again after a recording is added for the current day.

**Comment by ishanvaghani** [comment]
Yes, That is possible by scheduling next 30 notifications. As per my point of scheduling next 7 days notifications are enough because if user opens the app within 7 days then will again schedule it for next 7 days. Should i start on this?

**Comment by KyleKun** [comment]
> Yes, That is possible by scheduling next 30 notifications. As per my point of scheduling next 7 days notifications are enough because if user opens the app within 7 days then will again schedule it for next 7 days. Should i start on this?

Yes please, I think 2 weeks it's a good number to go with, 7 days may be too little for some users who still use the app but not everyday, so they don't forget about it easily cuz there's still notifications for a while.

**Comment by ishanvaghani** [comment]
Cool, I have started working on this

**Comment by ishanvaghani** [comment]
What is persistent notifications?

**Comment by KyleKun** [comment]
> What is persistent notifications?

You can't dismiss them from the notifications bar, they stay there all time.

**Comment by ishanvaghani** [comment]
Fixed this and raised the PR
